Source/Riff/Riff_Base.h

```diff
@@ -559,7 +560,7 @@ protected :
     //---------------------------------------------------------------------------
     //Read/Write
     virtual void    Read_Internal       ();
-    virtual void    Modify_Internal     ()                                      {}
+    virtual void    Modify_Internal     ();

```

Comment:
no need

Reply:
Definition is here: https://github.com/MediaArea/BWFMetaEdit/pull/283/files#diff-d451cf3c5401ca8dbca30e9caf315341c7ba564f909a93b642eb39e4c1e4c42aR650